### PR TITLE
Add method `to_d` (convert into BigDecimal) to some classes

### PR DESCRIPTION
**Pull request by takahashim**
Target classes: Integer, Float, String, BigDecimal, Rational, Complex and NilClass

### CODE REVIEW

File: stdlib/bigdecimal/0/big_decimal.rbs [hunk 1]
```diff
@@ -1187,6 +1187,18 @@ class BigDecimal < Numeric
   #
   def zero?: () -> bool
 
+  # call-seq:
+  #     a.to_d -> bigdecimal
+  #
```
Comment (ksss) [inline stdlib/bigdecimal/0/big_decimal.rbs:1192]: Maybe `call-seq` is not necessary.
I think you should use the `rbs annotate` tool.

**Review by ksss — COMMENTED** [review]
One way to do it is to create `bigdecimal-util` like `bigdecimal-math`, but RBS seems to be a tool that takes convenience over correctness, so this seems to be the way to go.

**Review by soutaro — DISMISSED** [review]
Could you try copying RDoc using `rbs annotate`?

**Review by soutaro — CHANGES_REQUESTED** [review]
Could you try copying RDoc using rbs annotate?

**Review by soutaro — APPROVED** [review]
Merging this PR. Will update the docs myself.

**Pull request merged**